library/extractor/src/main/java/com/google/android/exoplayer2/text/dvb/DvbTeletextDecoder.java

```diff
@@ -0,0 +1,36 @@
+package com.google.android.exoplayer2.text.dvb;
+
+import com.google.android.exoplayer2.text.SimpleSubtitleDecoder;
+import com.google.android.exoplayer2.text.Subtitle;
+import com.google.android.exoplayer2.text.SubtitleDecoderException;
+import com.google.android.exoplayer2.util.ParsableByteArray;
+import java.util.List;
+
+/** A {@link SimpleSubtitleDecoder} for DVB Teletext subtitles. */
+public final class DvbTeletextDecoder extends SimpleSubtitleDecoder {
+
+  private final DvbTeletextParser parser;
+
+  /**
+   * @param initializationData The initialization data for the decoder. The initialization data must
+   *     consist of a single byte array containing 2 bytes: magazine_number (1), page_number (2).
+   */
+  public DvbTeletextDecoder(List<byte[]> initializationData) {
+    super("DvbTeletextDecoder");
+    ParsableByteArray data = new ParsableByteArray(initializationData.get(0));
```

Comment:
nit: i think you can avoid this allocation by using guava's [`UnsignedInts.toInt`](https://guava.dev/releases/snapshot-jre/api/docs/com/google/common/primitives/UnsignedBytes.html#toInt(byte)) directly on the `initializationData` array.

Reply:
Done in cb97b5637c0906fb72a10a6ffc2e90f1db98da94